File: frontend/src/components/question.js
class Question {
    constructor(questionJSON) {
        this.id = questionJSON.id
        this.content = questionJSON.content
        this.ask = questionJSON.ask
        this.dont_ask = questionJSON.dont_ask
        this.user = questionJSON.user_id
    }

    questionHtml() {
        return `<p data-id="${this.id}">${this.content}</p>`
    }

}

File: backend/app/models/question.rb
class Question < ApplicationRecord
  belongs_to :user
  scope :ask, -> { where(ask: true) }
  scope :dont_ask, -> { where(dont_ask: true) }

  # def self.ask 
  #   where(ask: true)
  # end

  # def self.dont_ask 
  #   where(dont_ask: true)
  # end
end
